nixos/modules/system/boot/networkd.nix

```diff
@@ -95,8 +95,9 @@ let
     (assertOnlyFields [
       "Description" "DHCP" "DHCPServer" "IPForward" "IPMasquerade" "IPv4LL" "IPv4LLRoute"
       "LLMNR" "MulticastDNS" "Domains" "Bridge" "Bond" "IPv6PrivacyExtensions"
+      "IPv6AcceptRA" "LinkLocalAddressing"
     ])
-    (assertValueOneOf "DHCP" ["both" "none" "v4" "v6"])
+    (assertValueOneOf "DHCP" ["both" "none" "ipv4" "ipv6"])
```

Comment:
We should also change `both` to `yes` and `none` to `no` as it was introduced in systemd/systemd@cb9fc36a1211967e8c58b0502a26c42552ac8060. However according to this [code](https://github.com/systemd/systemd/blob/539ad37f18543d436ef9839e6bec20319ada53a2/src/network/networkd-network.c#L839-L849) the old style should still work.

Reply:
Done :)